layers/meta-resin-qemu/recipes-support/resin-runqemu/resin-runqemu.bb

```diff
@@ -0,0 +1,65 @@
+DESCRIPTION = "Resin helpers for running runqemu"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://runqemu-resin \
```

Comment:
What is the need of extending the runqemu scripts? Can you not select the needed arguments when using the standard runqemu scripts yocto provides?